src/chat/prompt_input.rs

```diff
@@ -1,19 +1,19 @@
 use makepad_widgets::*;
```

Comment:
"Large diffs are not rendered by default. " he... Of course, this file was the most involved with the issue.

But most work to support models here has been done in `entity_button.rs` (ex-`agent_button.rs`) and by `ChatEntityRef`. And some stuff has been simplified. For example, using `items()` from the list widget allows you directly know which item has been selected without reconstructing the whole list again.

So, actually, many changes here are just deleting the world `agent` with quick searches from many places like method names and so on.